yt/geometry/particle_geometry_handler.py

```diff
@@ -162,6 +162,25 @@ def _initialize_index(self):
         if not hasattr(self.ds, "_file_hash"):
             self.ds._file_hash = self._generate_hash()
 
+        # Load Morton index from file if provided
+        def _current_fname(order1, order2):
+            if getattr(ds, "index_filename", None) is None:
+                irange = f"[{order2}-{order2+2}]"
```

Comment:
It took me a moment to figure out we were creating an fstring for a glob, but this makes sense.